src/modules/category/category.service.js

```diff
@@ -0,0 +1,30 @@
+const Category = require('./category.model');
```

Comment:
Please, move all newly added logic for categories to another merge request to merge it in development as quickly as possible.